styleguide.md

```diff
@@ -21,7 +21,7 @@ Your project is likely to include multiple files, so you need to think about how
 **A readme.md file** should be included with every project. Learn to use [Markdown to format your documents](https://guides.github.com/features/mastering-markdown/).
 
 ## Variable and Function Naming
-How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite](https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
+How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite]((https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
```

Comment:
Good morning Prof. Wood:

got you, sorry for accidently made a commit on the styleguid.md.

Thanks,

Wenhui

On Fri, Jan 16, 2015 at 8:33 AM, Tim Wood <notifications@github.com> wrote:

> In styleguide.md
> <https://github.com/gwAdvNet2015/adv-net-samples/pull/26#discussion-diff-23079991>
> :
>
> > @@ -21,7 +21,7 @@ Your project is likely to include multiple files, so you need to think about how
> >  **A readme.md file** should be included with every project. Learn to use [Markdown to format your documents](https://guides.github.com/features/mastering-markdown/).
> >
> >  ## Variable and Function Naming
> > -How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite](https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
> > +How you name your variables, data structures, and functions can be the dominant factor on whether your code is readable. In general, we follow the [Composite]((https://github.com/gparmer/Composite/blob/master/doc/style_guide/composite_coding_style.pdf?raw=true) and [Google](http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Naming) naming conventions, although Google's recommendations are for C++ and thus use more capital letters than you are likely to use in a C program.
>
> You should undo this commit -- it looks like you have reverted a commit
> that I made which fixed a bug in the style file.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/gwAdvNet2015/adv-net-samples/pull/26/files#r23079991>.
>



-- 
V/R,

Wenhui Zhang

Email: wenhui@gwmail.gwu.edu
           Telephone: 1-(703) 424 3193